.github/build_real.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 set -e
-pip install .[testing]
+pip install -e .[testing]
```

Comment:
Why did you switch to the in-place installation?

Reply:
I was following Neil's instructions [here](https://github.com/mdolab/private-docs/issues/242). I think it's needed for codecov to work.

Reply:
Oh I see, makes sense, I guess Codecov does not access `site-packages`, I should have read that first